node/node.go

```diff
@@ -314,7 +314,11 @@ func onlyValidatorIsUs(state sm.State, privVal types.PrivValidator) bool {
 		return false
 	}
 	addr, _ := state.Validators.GetByIndex(0)
-	return bytes.Equal(privVal.GetPubKey().Address(), addr)
+	pv, err := privVal.GetPubKey()
+	if err != nil {
+		panic(err)
```

Comment:
we should strive to reduce the number of panics throughout the code.

Reply:
definitely agree with you here, but in the scenario that this function is called is if the node is the only validator. if it is not a validator, but the only node in the network then the node will panic anyways for nil validators. that is why I put the panic here. If you have a recommendation on how to change this let me know, not sure how to do it or if its needed.